linux/fredonia_linux/manifests/services.pp

```diff
@@ -1,3 +1,73 @@
 class fredonia_linux::services () {
-  #Filler
+  #services to manage 
+  #optional(if time allows): add notifications
+  # to update log when services are run & track
+  # for debugging 
+  #Samba running and enabled
+  package { 'samba':
+    ensure   => 'installed',
+    provider => 'apt',
```

Comment:
Ideally we should try to exclude the provider, as puppet can normally find which one to use. 

https://puppet.com/docs/puppet/latest/types/package.html#package-attribute-provider 
"You will seldom need to specify this — Puppet will usually discover the appropriate provider for your platform."